src/integrationTest/docker/docker-compose.yml

```diff
@@ -100,7 +107,26 @@ services:
       - "8082:8082"
     environment:
       KAFKA_REST_ZOOKEEPER_CONNECT: zookeeper-1:2181
-      KAFKA_REST_BOOTSTRAP_SERVERS: PLAINTEXT://kafka-1:9092,PLAINTEXT://kafka-2:9092,PLAINTEXT://kafka-3:9092
-      KAFKA_REST_LISTENERS: http://rest-proxy-1:8082
+      KAFKA_REST_LISTENERS: http://0.0.0.0:8082
       KAFKA_REST_SCHEMA_REGISTRY_URL: http://schema-registry-1:8081
       KAFKA_REST_HOST_NAME: rest-proxy-1
+      KAFKA_REST_COMPRESSION_TYPE: lz4
+
+  #---------------------------------------------------------------------------#
+  # Management Portal                                                         #
+  #---------------------------------------------------------------------------#
+  managementportal-app:
+    image: radarbase/management-portal:0.3.6
+    networks:
+      - kafka
+    ports:
+      - "8090:8090"
+    environment:
+      SERVER_PORT : 8090
+      SPRING_PROFILES_ACTIVE: dev
```

Comment:
I just have one question. I know that the `managementportal` context path was added to `prod` profile. I remember we also added it to dev profile. Is that the case? My MP knowledge is getting rusty. @yatharthranjan @blootsvoets . Otherwise the profile should be changed or the path should be altered at `gateway.yml`

Reply:
I don't remember as well. I think @dennyverbeeck could comment on this.